models/resultModel.js

```diff
@@ -0,0 +1,33 @@
+import mongoose from 'mongoose';
+
+const resultSchema = new mongoose.Schema({
+
+    recordId: { type: Array, unique: true, required: true },
```

Comment:
Please check how is array defined in mongoose. https://mongoosejs.com/docs/schematypes.html#arrays